var/spack/repos/builtin/packages/pmix/package.py

```diff
@@ -49,6 +49,12 @@ class Pmix(AutotoolsPackage):
     homepage = "https://pmix.github.io/pmix"
     url      = "https://github.com/pmix/pmix/releases/download/v2.0.1/pmix-2.0.1.tar.bz2"
 
+    version('3.0.2',    sha1='3a1c253cb078a9088e9a0a513f3a40e2f7f29cb8')
```

Comment:
we are using sha256 now, you can use the output from `spack checksum`

Reply:
Checks sums changed in new PR:
Update PMIX with releases 2.1.2-4 and 3.0.0-2 #9532
https://github.com/spack/spack/pull/9532